api/registry/management/commands/get_unmonitored_urls.py

```diff
@@ -1,45 +1,34 @@
-from django.core.management.base import BaseCommand, CommandError
-from django.conf import settings
+import json
+from collections import defaultdict
 from http.client import HTTPSConnection
 
-import json
-import re
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+from ninja.openapi.schema import OpenAPISchema
 
-# Will ignore e.g. anything starting with /admin/
-IGNORED_PATH_ROOTS = [
-    "admin",
-    "social",
-]
+from scorer.api import apis
 
-# Will ignore exact path matches
-IGNORED_URLS = [
-    "/registry/feature/openapi.json",
-    "/registry/feature/docs",
-    "/registry/feature/scorer/generic",
-]
+from .get_unmonitored_urls_config import get_config
 
-# This should only be used to get a release out quickly when necessary, after
-# which the URL should be added to the hardcoded IGNORED_URLS above to be
-# ignored in the future
-if settings.IGNORE_UNMONITORED_URLS:
-    IGNORED_URLS.extend(settings.IGNORE_UNMONITORED_URLS)
+http_method_map = {
+    "HEAD": 1,
+    "GET": 2,
+    "POST": 3,
+    "PUT": 4,
+    "PATCH": 5,
+    "DELETE": 6,
+    "OPTIONS": 7,
+}
 
 
 class Command(BaseCommand):
     help = "Removes stamp data and sets score to 0 for users in the provided list"
 
     def add_arguments(self, parser):
         parser.add_argument(
-            "--urls",
-            type=str,
-            help="""Local path to a file containing json output of `show_urls` command""",
-            required=True,
-        )
-        parser.add_argument(
-            "--out",
-            type=str,
-            help="Output file, json list of unmonitored paths",
-            required=True,
+            "--dry-run",
```

Comment:
Is this just for testing?

Reply:
No, this is to be used in the CI, where we do not want to create the monitors yet.
Typically we would want to create the monitors only when we deploy to prod.
But we would want to get errors in case we have missing configurations in the workflow that is triggered on pushes for example.